testing/perl-locale-msgfmt/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Automatically generated by apkbuild-cpan, template 2
+# Contributor: Timothy Legge <timlegge@gmail.com>
+# Maintainer: Timothy Legge <timlegge@gmail.com>
+pkgname=perl-locale-msgfmt
+_pkgreal=Locale-Msgfmt
+pkgver=0.15
+pkgrel=0
+pkgdesc="Compile .po files to .mo files"
+url="https://metacpan.org/release/Locale-Msgfmt/"
+arch="noarch"
+license="GPL-1.0-or-later Artistic-1.0-Perl"
+depends="$cpandepends"
```

Comment:
is effectively an empty variable since it depends on a variable that doesn't exist